draft-ietf-rats-architecture.md

```diff
@@ -844,40 +835,89 @@ a different format.
+
+## Implicit Timekeeping using Nonces
+
+The first approach places the onus of timekeeping solely on the remote
+entity (e.g., the Verifier for Evidence, or the Relying Party for an
+Attestation Result).
+This might be an appropriate choice in case the Attester does not have a reliable clock or time synchronisation is
+otherwise impaired.
+In this approach, a nonce is generated by a remote entity
 (e.g., the Verifier for Evidence, or the Relying Party for an Attestation
 Result), and the nonce is then signed and included along with the claims in
 the Evidence or Attestation Result, so that the remote entity knows that the
 claims were signed after the nonce was generated.
 
+## Explicit Timekeeping using Synchronized Clocks
+
 A second approach is to rely on synchronized clocks, and include a signed
-timestamp (e.g., using {{-rats-tuda}}) along with the
+timestamp along with the
 claims in the Evidence or Attestation Result, so that the remote entity knows
 that the claims were signed at that time, as long as it has some assurance that
-the timestamp is correct.  This typically requires additional claims about
+the timestamp is correct.
+This typically requires additional claims about
 the signer's time synchronization mechanism in order to provide such assurance.
-
-In either approach, it is important to note that the actual values in claims
-might have been generated long before the claims are signed.  If so, it is the
-signer's responsibility to ensure that the values are still correct when they
-are signed.  For example, values might have been generated at boot, and then
-used in claims as long as the signer can guarantee that they cannot have changed
+This allows associating a "rough" epoch to the Evidence or Attestation Result. 
+In this case, the epoch is said to be "rough" because 
+    a) it is always bound to the set of Claims as a whole, as opposed to having Claim being separately annotated, and 
+    b) it is not possible to distinguish between time of collection from the time of creation of the Claims set.
+
+## Interval Timekeeping using Nonces
+
+A third approach is a variant of the first; the difference is that it allows
+fine-grained epoch assignment.
+In this approach an Attester's signature spans the nonce as well as an
+Attester's trusted timetick counter(s). 
+Signatures across subsequent Evidence which embeds incrementing values of
+the trusted timetick counter(s) can be used to establish the time interval
+which has elasped since the nonce was received.
+
+## Distributed Interval Timekeeping using Local Clocks
+
+A fourth approach places the onus of timekeeping on all involved parties,
+but does not require the parties' clocks to be sychronized.
+A centralized entity is required to distribute non-repeatable values, which are referred to as
+Handles, to all involved parties in periodic intervals.
+Each Handle has a validity-span with an epoch that starts at the local time of
```

Comment:
The term "validity-span" is not a normal term as far as I know

Reply:
This text doesn't exist anymore.